http/1.10/modules/ROOT/pages/http-about-http-connector-migration.adoc

```diff
@@ -8,6 +8,7 @@ The Mule 4 version of Anypoint Connector for HTTP (HTTP Connector) contains the
 * The `http:request-config` element has a new inner connection element `http:request-connection`.
 * The HTTP body parameter replaces the source parameter in the request and is incorporated into the listener's response and error response.
 * The HTTP *Listener* source and HTTP *Request* operation return attributes.
+* The HTTP *Request* configuration has *Enabled Cookie* parameter set as `true` (default).
```

Comment:
The HTTP *Request* configuration has the *Enabled Cookie* parameter set to `true` (default).
```suggestion